grunttasks/serve.js

```diff
@@ -5,9 +5,15 @@
 module.exports = function (grunt) {
   'use strict';
 
-  grunt.registerTask('serve', [
-    'build:development',
-    'hapi',
-    'watch'
-  ]);
+  grunt.registerTask('serve', 'Builds the dist assets, starts the Hapi server, and watches for changes.', function (target) {
+    if (!target) {
+      target = 'development';
+    }
```

Comment:
The `target` argument gets set when you call `grunt serve:production` (where `serve` is the "task" and `production` is the "target").
So the logic is saying "if there is no `target` specified" (e.g. `grunt serve`) then set a default value of "development", whereas if the `target` _was_ specified (e.g. `grunt serve:development` or `grunt serve:production`) then use the specified target.

Or something like that.